socs/agents/meinberg_syncbox/agent.py

```diff
@@ -280,7 +303,7 @@ def acq(self, session, params=None):
                 {'status': 0,
                  'description': 'tai'},
             'mbgSyncboxN2XPtpUTCOffset_0':
-                {'status': '37 sec',
+                {'status': '37000000',
```

Comment:
These all won't be strings anymore, right?
```suggestion
                {'status': 37000000,
```

Also, the `field_name` modification would make this `'mbgSyncboxN2XPtpUTCOffset_0_value'` now too, right?